tests/call-centre/Dockerfile

```diff
@@ -15,9 +15,10 @@ RUN groupadd -r pptruser && useradd -r -g pptruser -G audio,video pptruser \
 # Run everything after as non-privileged user.
 USER pptruser
 
-COPY . ./
+COPY protractor.conf.js protractor.constants.js ./
+COPY test ./test
```

Comment:
_nitpick_
💭 If we only need these files, it might be an alternative to setup a `.dockerignore` file.

Reply:
I've had trouble getting the `.dockerignore` file working. Either I'm doing something wrong or it doesn't play nicely with subdirectories. 

May be I need a `.dockerignore` for each folder where there's a `Dockerfile`, rather than in the root, where the `docker-compose.yml` lives.

I won't fix this now - one for another day.